### DISCUSSION TITLE
Using crosh to disable ext

### DISCUSSION THREAD

**Discussion by cooltoo37** [3kh0/ext-remover#574] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there a way to use crosh to disable apps or extensions on chrome versions 102

**Comment by GrinchERR** [comment, marked as answer]
Crosh is useless unless you are in DEV mode. I don't think its possible and it would not be worth trying to figure out how if it is. If javascript it blocked i would just recommend using Killcurly. Admins cant easily find out. The warnings it lists are not common. It sometimes randomly disables. If you just click power and refresh to restart your computer the extensions will go back to working like normal.

**Reply by cooltoo37** [reply]
no i have everything unblocked javascript corsh and bookmarklets just when i am at school the cisco umbrella app blocks me from playing games and none of the exploits disable apps

**Reply by GrinchERR** [reply]
It’s because of school network restrictions. Just get a vpn. Or host one urself

**Reply by amelia-bartlett** [reply]
> no i have everything unblocked javascript corsh and bookmarklets just when i am at school the cisco umbrella app blocks me from playing games and none of the exploits disable apps

cisco umbrella is network level - you need a vpn or ssh tunnel

**Reply by cooltoo37** [reply]
yo can i get any vpns becuase I am dumb kinda and cant find any pls

**Comment by GrinchERR** [comment]
﻿Can you send it to me too. I friend requested you or you could just dm me GrinchERR#6763
